### DISCUSSION TITLE
A lot of `The unknown error was caught at path: /_next/static/css/` errors caught in Sentry

### DISCUSSION THREAD

**Discussion by angelod1as** [vercel/next.js#66412] [Help]
### Summary

I have a NextJS project that has Sentry and Datadog implemented for logging and tracing.

There are _lots_ of errors that are always like this:

`The unknown error was caught at path: /_next/static/css/8346b44384d80ec4.css`

Is there a way to know better why these errors happen?

The production application works fine, the CSS is correctly loaded, all is good. But the errors remain.

### Additional information

```js
Operating System:
  Platform: darwin
  Arch: arm64
  Version: Darwin Kernel Version 23.4.0: Fri Mar 15 00:10:42 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6000
  Available memory (MB): 32768
  Available CPU cores: 10
Binaries:
  Node: 20.12.2
  npm: 9.6.7
  Yarn: 1.22.21
  pnpm: 8.15.4
Relevant Packages:
  next: 14.2.3 // Latest available version is detected (14.2.3).
  eslint-config-next: 14.2.3
  react: 18.3.1
  react-dom: 18.3.1
  typescript: 5.4.5
Next.js Config:
  output: N/A
```

**Comment by icyJoseph** [comment]
Hi,

I wonder, if by any chance, those CSS paths are being loaded as `script` on your app? I saw a couple of weeks ago, someone whose WebPack config, had created script tags pointing to CSS files, which in turn causes a bunch of errors.

**Reply by angelod1as** [reply]
Hmmm, that's a good question. We're using the [native SCSS support](https://nextjs.org/docs/app/building-your-application/styling/sass), so there's no extra configuration done.
